src/AddressSpace.hpp

```diff
@@ -14,7 +14,9 @@
 #define __ADDRESSSPACE_HPP__
 
 #include <stdint.h>
+#ifndef RUST_SGX
 #include <stdio.h>
```

Comment:
This was added mostly due to stderr. Also we redefine fprintf to our own function.
But I can try without removing stdout too, it might make it a rule to include UnwindRustSgx.h to be included after stdio.h.